src/microbithal_js.c

```diff
@@ -431,8 +451,8 @@ bool microbit_hal_audio_is_expression_active(void) {
     return mp_js_hal_audio_is_expression_active();
 }
 
-void microbit_hal_audio_play_expression_by_name(const char *name) {
-    mp_js_hal_audio_play_expression_by_name(name);
+void microbit_hal_audio_play_expression(const char *name) {
```

Comment:
`*name` was also renamed to `*expr`.

Reply:
Aligned in 70c57d7c6b5ad733574beffdb8b71a21e1e168d3. Shouldn't have had any functional difference but makes sense to reduce the diff.